content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,370 @@
+
+=======================
+Accounting Localization
+=======================
+
+.. warning::
+
+    This tutorial requires knowledges about how to build a module in Odoo (see
+    :doc:`/developer/howtos/backend`).
+
+
+How localization works
+======================
+
+On installing the `account <https://github.com/odoo/odoo/tree/15.0/addons/account>`__ module, the localization module corresponding to the country code of the company is installed automatically.
+In case of no country code set or no localization module found, the `l10n_generic_coa <https://github.com/odoo/odoo/tree/15.0/addons/l10n_generic_coa>`__ (US) localization module is installed by default.
+Check `post init hook <https://github.com/odoo/odoo/blob/15.0/addons/account/__init__.py>`__ for details.
+
+For example, `l10n_ch <https://github.com/odoo/odoo/tree/15.0/addons/l10n_ch>`__ will be installed if the company has ``Switzerland`` as country.
+
+Building a localization module
+==============================
+
+Structure of a basic ``l10n_XX`` module may be described as following ``__manifest__.py`` file:
+
+.. code-block:: py
+
+    {
+        "name": "COUNTRY - Accounting",
+        "version": "1.0.0",
+        "category": "Accounting/Localizations/Account Charts",
+        "license": "LGPL-3",
+        "depends": [
+            "account",
+            # "l10n_multilang",
+        ],
+        "data": [
+            # Chart of Accounts
+            "data/account_chart_template_data.xml",
+            "data/account_account_tag_data.xml",
+            "data/account.account.template.csv",
+            "data/account.group.template.csv",
+
+            # Taxes
+            "data/account_tax_group_data.xml",
+            "data/account_tax_report_data.xml",
+            "data/account_tax_template_data.xml",
+            "data/account_fiscal_position_template_data.xml",
+            "data/account_account_template_post_data.xml",
+
+            "data/account_chart_post_data.xml",
+            "data/account_chart_template_try_loading.xml",
+
+            # Views and others
+            "views/xxxmodel_views.xml"
+        ],
+        "demo": [
+            "demo/demo_company.xml",
+        ]
+    }
+
+
+In the first file `data/account_chart_template_data.xml` we give name for the Chart of Accounts and set some basic fields.
+
+.. seealso::
+   :ref:`Chart Template References <reference/account_chart_template>`
+
+.. example::
+  `addons/l10n_ch/data/l10n_ch_chart_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/l10n_ch_chart_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/l10n_ch_chart_data.xml
+    :language: xml
+    :start-at: l10nch_chart_template
+    :end-at: </record>
+
+
+.. note::
+
+  Recommended **xmlid** for the record is `chart_template`.
```

Comment:
Funny that the example above doesn't respect that :p Should we even bother recommanding anything here ? It actually makes no difference from a technical point of view.

Reply:
It is annoying to have `l10n_xx.l10nxx_chart_template`. (quite repetitive) Would be easier if it was uniform.